src/mina-signer/index.d.ts

```diff
@@ -1,6 +1,7 @@
 // this file is a wrapper for supporting types in both commonjs and esm projects
 
 import Client from './mina-signer.ts';
-export { type NetworkId } from './src/types.ts';
+import { NetworkId } from './src/types.ts';
 
-export = Client;
+export default Client;
```

Comment:
was this change necessary?

Reply:
It does not impact functionality but since mina-signer is ESM project it removes error on that line if used to target CJS.